spec.bs

```diff
@@ -2603,6 +2644,9 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
   : <dfn for="source snapshot params">initiator fenced frame config instance</dfn>
   :: a [=fenced frame config instance=] or null, initially null.
 
+  : <dfn for="source snapshot params">initiator ancestor root origin</dfn>
+  :: an [=origin=] or null, initially null.
```

Comment:
Let's add a Note about what this actually is in practice.

Reply:
There's a note further down that explains this field along with `initiator fenced frame config instance` and `target fenced frame config`. I've moved that block up to be closer to these fields.